.coveragerc

```diff
@@ -12,6 +12,7 @@ ignore_errors = True
 omit =
     conftest.py
     setup.py
+    docs/*
     jwst/conftest.py
     jwst/tests/*
```

Comment:
May want to add `tests_nightly` as well.

Reply:
Ah, good catch